include/seqan3/search/fm_index/fm_index_cursor.hpp

```diff
@@ -268,10 +262,12 @@ class fm_index_cursor
      * No-throw guarantee.
      */
     template <Alphabet char_t>
+    //!\cond
+        requires ImplicitlyConvertibleTo<char_t, typename index_t::char_type>
```

Comment:
See above. Also I don't think you even need to require alphabet. Convertbility is all you need,